template/.vscode/settings.json

```diff
@@ -30,5 +30,6 @@
   // Linting:
   "eslint.enable": true,
   "eslint.validate": ["javascript", "javascriptreact", "vue"],
-  "stylelint.enable": true
+  "stylelint.enable": true,
+  "javascript.validate.enable": false
```

Comment:
Could you please move this line into `// Disables default checks:` section?

Reply:
Sorry, but I have meant to move this line into the existing section: https://github.com/kirillvy/wemake-vue-template/blob/d2d99f96839055e05cdb3dfc053658b92929acda/template/.vscode/settings.json#L27-L28

Not to create a new one.